config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/AnnotableEntity.java

```diff
@@ -50,6 +50,7 @@
             AnnotatedEntity<GatewayEntity> annotatedEntity = new AnnotatedEntity(this);
             annotations.forEach(annotation -> {
                 if (ANNOTATION_TYPE_BUNDLE.equalsIgnoreCase(annotation.getType())) {
+                    annotatedEntity.setId(annotation.getId());
```

Comment:
currently we are using this id for entity, we should have separate field for bundle id

Reply:
Yes correct, we are addressing that in other story, i will revert this change